### DISCUSSION TITLE
FP 10.4 11: filters update missing

### DISCUSSION THREAD

**Discussion by Illegit** [freeplane/freeplane#597] [Help]
Filter composer is loading my filters without problem on one pc, but on another pc I'm not able to read it, the filters are empty: 

- I even copied the same autofilter file working in the other pc: no result the filters were still empty (even after reloading FP)
- I reinstalled FP, but no result, always the same message "no named filters"

Any suggestion? I do not think it is a bug... but.
Thanks

**Comment by dpolivaev** [comment]
Open a bug report and attach the autofilter file and logs

**Reply by Illegit** [reply]
ok
